test/keysym.c

```diff
@@ -27,6 +27,9 @@
 #include "test.h"
 #include "keysym.h" /* For unexported is_lower/upper/keypad() */
 
+/* Needed because XKB_KEYSYM_MAX has parentheses */
+#define KEYSYM_MAX 0x1fffffff
```

Comment:
tbh I think it'd be better off using the actual strings in the tests instead of having a separate `#define` here. e.g
```c
    assert(test_string("0x1fffffff", XKB_KEYSYM_MAX));
```
is immediately understandable without the extra redirection.

Reply:
I dropped the parentheses so we can use XKB_KEYSYM_MAX for clarity.